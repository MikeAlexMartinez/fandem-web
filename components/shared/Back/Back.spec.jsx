import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import BackComponent from './Back';

describe('<Back />', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(<BackComponent />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
