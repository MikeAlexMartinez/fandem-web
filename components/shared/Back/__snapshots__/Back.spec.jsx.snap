// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Back /> renders and matches the snapshot 1`] = `
<Back
  classes={
    Object {
      "link": "Back-link-1",
      "text": "Back-text-2",
    }
  }
/>
`;
